ludwig/automl/utils.py

```diff
@@ -78,6 +78,23 @@ def avg_num_tokens(field: Series) -> int:
     return avg_words
 
 
+def check_if_boolean(field: Series) -> bool:
+    if len(field) > 5000:
+        field = field.sample(n=5000, random_state=40)
```

Comment:
I believe this function could be given a Dask Series, which does not support `sample(n)`. Dask only supports sampling a fraction of the data. Typically we would just use `field.head(5000)` here, and avoid calling `len`, which is also expensive in Dask.

More general question: do we need to compute this directly from the series when we already have a function called `get_distinct_values` that returns the top k distinct values? We should be able to derive this info from the distinct values instead of recomputing it.

Reply:
Makes sense. Changed it accordingly